sdks/python/apache_beam/typehints/schemas.py

```diff
@@ -587,6 +588,7 @@ def to_language_type(value):
   def register_logical_type(cls, logical_type_cls):
     """Register an implementation of LogicalType."""
     cls._known_logical_types.add(logical_type_cls.urn(), logical_type_cls)
+    return logical_type_cls
```

Comment:
These place were some minor bugs. If not return logical_type_cls, class decorated with `@register_logical_type` cannot be imported (will be None) and docstring is also missing from pydoc; signatures of `to_representation_type` and `to_language_type` should have a value parameter as its child classes. Fixed here.

Reply:
Thank you!